addons/iot_drivers/tools/helpers.py

```diff
@@ -640,3 +648,32 @@ def toggle_remote_connection(token=""):
         )
         return True
     return False
+
+
+def check_network(host=None):
+    host = host or get_gateway()
+    if not host:
+        return None
+
+    packet_loss, avg_latency = mtr(host)
+
+    if packet_loss is None or packet_loss >= 50 or avg_latency is None:
+        return "unreachable"
+    if avg_latency < 50 and packet_loss < 1:
+        return "fast"
+    if avg_latency < 150:
+        return "normal"
+    return "slow"
```

Comment:
the criteria of speed should be different for the router and the global network speed.
speed of 149ms can be acceptable for the global network but 149 to reach the router is very slow